src/resources/reviews.jsx

```diff
@@ -0,0 +1,84 @@
+import { 
+    List, 
+    ReferenceField,
+    TextField, 
+    Datagrid,
+    Edit,
+    SimpleForm,
+    ReferenceInput,
+    TextInput,
+    DateField,
+    DateTimeInput,
+    Create,
+    NumberField,
+    NumberInput,
+    Show,
+    SimpleShowLayout
+} from "react-admin";
+
+const reviewFilters = [
+  <TextInput source="q" label="Search" alwaysOn />,
+];
+
+export const ReviewList = () => (
```

Comment:
Would the category ratings not fit in anymore?

Reply:
i thought there are not nessecary here

Reply:
I don't think so either